dessia_common/workflow/core.py

```diff
@@ -101,15 +101,44 @@ def dict_to_object(cls, dict_: JsonSerializable, force_generic: bool = False,
     def copy(self, deep: bool = False, memo=None):
         """ Copies and gives a new object with no linked data. """
         return TypedVariable(type_=self.type_, name=self.name)
+    
+    def get_script_elements(self, script: ToScriptElement) -> ToScriptElement:
+        """
+        Update elements of the script representing the workflow.
+
+        return ToSriptElement
+        """
+        types = ["Dict", "Tuple"]
+        try:
+            script.declaration += f", type_={self.type_.__name__}"
+        except AttributeError:
+            if is_typing(self.type_):
+                args = self.type_.__args__
+                if len(args) == 1:
+                    args = self.type_.__args__[0].__name__
+                elif len(args) == 2:
+                    args = f"{args[0].__name__}, {args[0].__name__}"
+                script.declaration += f", type_={serialize_typing(self.type_).split('[')[0]}" \
+                                      f"[{args}]"
+                script.imports.append(f"{self.type_.__args__[0].__module__}.{self.type_.__args__[0].__name__}")
```

Comment:
I don't understand this. Why do we take only index 0 ?

Reply:
because `self.type__.__args__ ` is a tuple (ex :` (dessia_common.files.BinaryFile,)`)